### DISCUSSION TITLE
How to add the AG Grid Enterprise registration code?

### DISCUSSION THREAD

**Discussion by zbxiany** [zauberzeug/nicegui#3048] [Q&A]
### Question

Who can help me？
How to add the registration code to the NiceGUI file when I have registered for the AG Grid Enterprise version

<img src="https://github.com/zauberzeug/nicegui/assets/86754894/fa526d07-ef7e-4cdf-905d-226c4175da4b" width="500">

**Comment by falkoschindler** [comment]
Hi @zbxiany, this question has been answered in discussion #2431. Currently AG Grid Enterprise isn't supported. 😕

**Reply by zbxiany** [reply]
I just want to know which file is more suitable for placing the registration code in, and I don't care if the enterprise version can be used

**Reply by falkoschindler** [reply]
Sorry, I don't understand the point in adding a registration code for an unsupported version.
Anyway, there is no `@ag-grid-enterprise` file or module in the NiceGUI codebase, so you can't import the license manager from it. Maybe it's possible by adding the file yourself, but that's beyond what NiceGUI currently supports.

**Reply by zbxiany** [reply]
I just tried replacing the community version with the enterprise version JS, and many advanced features can be used
